org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/core/ReplModule.java

```diff
@@ -25,9 +26,11 @@
 import com.google.inject.multibindings.MapBinder;
 
 /**
- * Client library bindings.
+ * This class binds the core classes. It is intended to be subclassed by client implementations.
+ * These subclasses should bind their implementations of {@link IRepl}, {@link IMessageHook} and
+ * {@link IResultHook}.
  */
-public class ReplModule extends SpoofaxModule {
+public abstract class ReplModule extends SpoofaxModule {
```

Comment:
Dummies are only useful if they serve any purpose at all, which I think in this case they wouldn't.